On Windows, the enum.inc file could not be found.

## Description

### Current State
```
#include <stdio.h>
#include <string.h>
#include <wasmedge/wasmedge.h>

int main() {
    /* Create the configure context and add the WASI support. */
    /* This step is not necessary unless you need WASI support. */
    WasmEdge_ConfigureContext *ConfCxt = WasmEdge_ConfigureCreate();
    WasmEdge_ConfigureAddHostRegistration(ConfCxt, WasmEdge_HostRegistration_Wasi);

    /* The configure and store context to the VM creation can be NULL. */
    WasmEdge_VMContext *VMCxt = WasmEdge_VMCreate(ConfCxt, NULL);


    /* Create the import object. */
    WasmEdge_String importName = WasmEdge_StringCreateByCString("env");
    WasmEdge_ImportObjectContext *ImpObj = WasmEdge_ImportObjectCreate(importName);
    WasmEdge_StringDelete(importName);


    WasmEdge_VMRegisterModuleFromImport(VMCxt, ImpObj);

    /* Run the WASM function from file. */
    //WasmEdge_String FuncName1 = WasmEdge_StringCreateByCString("print_input");
    WasmEdge_String FuncName = NULL;
    WasmEdge_Result Res = WasmEdge_VMRunWasmFromFile(VMCxt, "test.wasm", FuncName, NULL, 0, NULL, 0);

    if (WasmEdge_ResultOK(Res)) {
        printf("Successfully executed !");
    } else {
        printf("Error !");
    }

    /* Resources deallocations. */
    WasmEdge_VMDelete(VMCxt);
    WasmEdge_ConfigureDelete(ConfCxt);
    return 0;
}
```
When I compiled the above source code, the following error occurred.
```
fatal error C1083:  “enum.inc”: No such file or directory
```

## Environment
- Hardware Architecture: x86
- Operating system: windows11

--- a/.github/scripts/wasmedge.wxs
+++ b/.github/scripts/wasmedge.wxs
@@ -44,6 +44,9 @@
 			<Component Id="enum_types.h" Guid="*">
 				<File Id="enum_types.h" Source="build\include\api\wasmedge\enum_types.h" KeyPath="yes" Checksum="yes"/>
 			</Component>
+			<Component Id="enum.inc" Guid="*">
+				<File Id="enum.inc" Source="build\include\api\wasmedge\enum.inc" KeyPath="yes" Checksum="yes"/>
+			</Component>
 			<Component Id="int128.h" Guid="*">
 				<File Id="int128.h" Source="build\include\api\wasmedge\int128.h" KeyPath="yes" Checksum="yes"/>
 			</Component>
@@ -65,6 +68,7 @@
 			<ComponentRef Id="wasmedge.exe" />
 			<ComponentRef Id="wasmedgec.exe" />
 			<ComponentRef Id="wasmedge_c.dll" />
+			<ComponentRef Id="enum.inc" />
 			<ComponentRef Id="enum_configure.h" />
 			<ComponentRef Id="enum_errcode.h" />
 			<ComponentRef Id="enum_types.h" />

